ledger-tool/src/main.rs

```diff
@@ -1081,7 +1081,7 @@ fn load_bank_forks(
             exit(1);
         }
         account_paths.split(',').map(PathBuf::from).collect()
-    } else if blockstore.is_primary_access() {
+    } else if blockstore.is_primary_access() || arg_snapshot_from_file {
```

Comment:
I'm concerned about this change. If someone runs ledger-tool while the validator is running, this could lead to deletion of appendvec files in the active validator accounts directory.

Where are the hardlink files? Do we need to use the `accounts` directory directly?